src/libraries/ParseReturn.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.20;
+
+library ParseReturn {
```

Comment:
maybe 1 comment here saying that all hooks return
`bytes4`, or `bytes4, 32-byte-delta`, or `bytes4, 32-byte-delta, uint24`